test/ERC20.ts

```diff
@@ -2,3 +2,8 @@ export const NON_STANDARD_ERC20 = [
   "function transfer(address recipient, uint256 amount)",
   "function transferFrom(address sender, address recipient, uint256 amount)",
 ];
+
+export const ERC20_RETURNING_UINT = [
```

Comment:
This is cute, is this a minimal ABI for the SafeERC20 Library?

Reply:
Its a "fake" ABI for a non-standard ERC20 that returns a `uint256` success value instead of a bool. For testing.